services/app-api/handlers/reports/fetch.ts

```diff
@@ -17,31 +17,33 @@ export const fetchReport = handler(async (event, _context) => {
     TableName: process.env.MCPAR_REPORT_TABLE_NAME!,
     Key: { state, id: reportId },
   };
-  const response = await dynamoDb.get(reportMetadataParams);
-  const reportMetadata: any = response.Item; // TODO: strict typing
-  const { formTemplateId, fieldDataId } = reportMetadata;
+  try {
+    const response = await dynamoDb.get(reportMetadataParams);
+    if (!response?.Item) throw "Record not found in database";
```

Comment:
Might want to move these to the `errors.ts` file and pull them in.

Reply:
👍 Good call.

Reply:
completed